x-pack/platform/plugins/private/file_upload/server/import_data.ts

```diff
@@ -40,7 +41,14 @@ export function importDataProvider({ asCurrentUser }: IScopedClusterClient) {
         createdIndex = index;
 
         // create the pipeline if one has been supplied
-        if (pipelineId !== undefined) {
+        if (createPipelines !== undefined) {
+          for (let i = 0; i < createPipelines.length; i++) {
+            const resp = await createPipeline(createPipelines[i].id, createPipelines[i].pipeline);
```

Comment:
```suggestion
          for (let {id, pipeline} of createPipelines) {
            const resp = await createPipeline(id, pipeline);
```

Reply:
Updated in https://github.com/elastic/kibana/pull/206864/commits/8bf59f8a149da53e5f73aa55cf472bf9df13c11b